__tests__/endpoint.test.js

```diff
@@ -34,38 +34,43 @@ describe("/api/topics", () => {
         ]);
       });
   });
-  test("status:404, responds with an error message when passed a bad path", () => {
+  test("status:400, responds with an error message when passed a bad path", () => {
     return request(app)
       .get("/api/nvnewbs")
       .expect(404)
       .then(({ body }) => {
-        expect(body.msg).toBe("Not Found");
+        console.log(body, "<--- body");
+        expect(body.msg).toBe("bad request");
       });
   });
 });
-describe("/api", () => {
-  test("/api responds by creating a file matching that of the example.json file", () => {
+describe("/api/articles/", () => {
+  test("200: responds with articles related to ID", () => {
     return request(app)
-      .get(`/api`)
+      .get("/api/articles/1")
       .expect(200)
       .then(({ body }) => {
-        expect(body.API).toMatchObject(endpointJSON);
+        expect(body).toMatchObject([
```

Comment:
Nice use of toMatchObject here - this will allow you flexibility to extend your endpoint at a later date